features/call/api/src/main/kotlin/io/element/android/features/call/api/CurrentCallObserver.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ * Please see LICENSE in the repository root for full details.
+ */
+
+package io.element.android.features.call.api
+
+import kotlinx.coroutines.flow.StateFlow
+
+interface CurrentCallObserver {
```

Comment:
The name is a bit weird, the observer is the one who subscribe to the flow usually, not the one providing.
I'd call this CurrentCallService or CurrentCallDataSource for exemple

Reply:
2b5acb30232d026fc71f9e7e00a3c228ce370cb3